src/js/node/http.ts

```diff
@@ -766,6 +766,7 @@ async function consumeStream(self, reader: ReadableStreamDefaultReader) {
     if (self[abortedSymbol]) return;
     if (done) {
       self.push(null);
+      self.complete = true;
```

Comment:
This needs to be before `push(null)` to be consistent with Node.js:

https://github.com/nodejs/node/blob/78a326e2eb236874c45a7937d51e5d3ab3c91a3e/lib/_http_common.js#L141-L153

Reply:
fixed in https://github.com/oven-sh/bun/pull/11926/commits/5205985edadd0b7ed62f2feb17b54accda755916